api/internal/target/kusttarget_configplugin.go

```diff
@@ -241,6 +241,13 @@ var transformerConfigurators = map[builtinhelpers.BuiltinPluginType]func(
 			if label.IncludeSelectors {
 				fss, err = fss.MergeAll(tc.CommonLabels)
 			} else {
+				// merge spec/template/metadata fieldSpec if includeTemplate flag is true
+				if label.IncludeTemplates {
+					fss, err = fss.MergeOne(types.FieldSpec{Path: "spec/template/metadata/labels", CreateIfNotPresent: false})
+					if err != nil {
+						return nil, err
+					}
+				}
```

Comment:
It won't let me comment on it, but the linter is complaining about line 292 below. You can either wrap the error or add `//nolint:wrapcheck` to the end of the line.

Reply:
wrap added

Reply:
seems there was also a failure downloading a module [here](https://github.com/kubernetes-sigs/kustomize/runs/6016606223?check_suite_focus=true), can we re-run?

Reply:
I don't know why the line number is wrong, but I think it is actually complaining about L249. It shouldn't complain about existing code, and the signature of the error in question is `func (sigs.k8s.io/kustomize/api/types.FsSlice).MergeOne(x sigs.k8s.io/kustomize/api/types.FieldSpec) (sigs.k8s.io/kustomize/api/types.FsSlice, error)`. 

Regardless, please rebase the PR. We've made major improvements to the linter config since the commit it is current based on, and I'd like to make sure this runs with the latest config before merging.

Reply:
PR is rebased, I wrapped the error in that line and ran the unit tests locally, can we re-run? 🙏